cliquet/resource/__init__.py

```diff
@@ -1028,13 +1028,13 @@ def process_record(self, new, old=None):
 
         return annotated
 
-    def postprocess(self, result, action=ACTIONS.READ):
+    def postprocess(self, result, **kwargs):
```

Comment:
:) Since I am given the absolute power to decide that, I will say no and push the big green button :D